src/com/amplitude/api/DatabaseHelper.java

```diff
@@ -22,10 +22,16 @@
     static DatabaseHelper instance;
     private static final String TAG = "com.amplitude.api.DatabaseHelper";
 
+    private static final String AMPLITUDE_STORE_TABLE_NAME = "amplitudestore";
```

Comment:
how about just "store" or "values"? no need for Amplitude since the database is Amplitude specific